pkg/minikube/reason/known_issues.go

```diff
@@ -1053,6 +1053,15 @@ var guestIssues = []match{
 		},
 		Regexp: re(`'/var/lib/dpkg': No such file or directory`),
 	},
+	{
```

Comment:
how about in addition to this ( as a fall back ) we set the feature gate automatically for the user?

Reply:
we have example of it in cni.go

```
		// for containerd and docker: auto-set custom CNI via kubelet's 'cni-conf-dir' param, if not user-specified
		eo := fmt.Sprintf("kubelet.cni-conf-dir=%s", CustomConfDir)
		if !cc.KubernetesConfig.ExtraOptions.Exists(eo) {
			klog.Infof("auto-setting extra-config to %q", eo)
			if err := cc.KubernetesConfig.ExtraOptions.Set(eo); err != nil {
				return fmt.Errorf("failed auto-setting extra-config %q: %v", eo, err)
			}
```

Reply:
Added!